### PR TITLE
DOM II - Working with event listeners

### PR DESCRIPTION
**Pull request by Rexben001**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1 +1,109 @@
...
+
+//click
+bodyPage.addEventListener('click', (event) => {
+    if (event.srcElement.className === 'nav-link' || event.srcElement.localName === 'img')
+    {
+        return;
+    } else if (event.srcElement.className === 'btn' || event.srcElement.className === 'destination')
+    {
+        return;
+    }
```
Comment (GianBarGian) [inline js/index.js]: Why break this in multiple blocks of code?

File: js/index.js [hunk 2]
```diff
@@ -1 +1,109 @@
...
+    {
+        event.target.style.display = 'none';
+    }
+});
+
+//mouseleave
+
+const link = document.getElementsByTagName('a');
+
+for (let i of link)
```
Comment (GianBarGian) [inline js/index.js]: This is fine, you could as well use query selectors and use arrays methods if you prefer.

**Comment by GianBarGian** [comment]
Your website is really good. Every event that trigger a function works properly and it's really cool to watch at.
You have just a small bug with your fixed nav bar, when you trigger an animation it overflows it.
You could want to modify his z-index.

Not too much to add, your code is clear and well structured. 
I would say if you check out Green Sock animations, you could make a good use of it.
Good job!

File: js/index.js (hunk 1 above)
Comment (Rexben001) [reply js/index.js]: I just feel it'd be neater that way

**Pull request merged**